### PR TITLE
Gp2489

### PR DESCRIPTION
**Pull request by pruzanov**
Updated sequenza: summary plot, additional png panels with contour and genome view plots to illustrate the effect of gamma parameter

### CODE REVIEW

**Review by albertojleon — APPROVED** [review]

**Comment by pruzanov** [comment]
retest this please

File: sequenza.wdl [hunk 1]
```diff
@@ -190,41 +210,68 @@ command <<<
...
+   gamma = gammas[g]
+   os.system("mkdir -p gammas/" + gamma)
+   os.system("unzip " + zips[g] + " -d gammas/" + gamma + "/")
+   chunk = {
+     'cellularity': [],
+     'ploidy': [],
+     'SLPP': []
+   }
+   if not isfile(paths[g]):
+     Gcontinue
```
Comment (lheisler) [inline sequenza.wdl:222]: is this a typo?

Comment (lheisler) [inline sequenza.wdl:222]: Is this a typo?

**Review by pruzanov — COMMENTED** [review]
corrected

**Comment by pruzanov** [comment]
I removed Jonathon from reviewers (he is away at the moment, I think) but I would like him to take a look at this sometime in a future

**Review by lheisler — APPROVED** [review]

**Review by mlaszloffy — APPROVED** [review]

**Pull request merged**